packages/communication-react/CHANGELOG.stable.md

```diff
@@ -7,36 +7,32 @@
 ## [1.22.0](https://github.com/azure/communication-ui-library/tree/1.22.0)
 
 Mon, 16 Dec 2024 19:44:14 GMT 
-[Compare changes](https://github.com/azure/communication-ui-library/compare/1.22.0-beta.1...1.22.0)
+[Compare changes](https://github.com/azure/communication-ui-library/compare/1.21.0...1.22.0)
 
-### Features
-- Merge branch 'main' into fuyan/hard_mute_part3_integrate_web_sdk ([PR #5437](https://github.com/azure/communication-ui-library/pull/5437) by fuyan@microsoft.com)
 ### Improvements
-- nanoid package update ([PR #5490](https://github.com/azure/communication-ui-library/pull/5490) by 98852890+vhuseinova-msft@users.noreply.github.com)
-- Automate dominant speaker, pinning, and spotlight tests ([PR #5173](https://github.com/azure/communication-ui-library/pull/5173) by miguelgamis@microsoft.com)
+- Update nanoid package ([PR #5490](https://github.com/azure/communication-ui-library/pull/5490) by 98852890+vhuseinova-msft@users.noreply.github.com)
 - Update `@azure/communication-calling-effects` to 1.1.2 ([PR #5407](https://github.com/azure/communication-ui-library/pull/5407) by 2684369+JamesBurnside@users.noreply.github.com)
-- Update @azure/communication-calling to 1.31.2-beta.2 ([PR #5440](https://github.com/azure/communication-ui-library/pull/5440) by 79475487+mgamis-msft@users.noreply.github.com)
-- Add hard mute localization ([PR #5485](https://github.com/azure/communication-ui-library/pull/5485) by fuyan@microsoft.com)
-- Build captions inside caling stateful sample ([PR #5502](https://github.com/azure/communication-ui-library/pull/5502) by 96077406+carocao-msft@users.noreply.github.com)
+- Make Start Captions Button a public component ([PR #5499](https://github.com/azure/communication-ui-library/pull/5499) by 96077406+carocao-msft@users.noreply.github.com)
```

Comment:
I wonder if these captions items are a feature we can call out? new component that provides a lot of cool functionality? thoughts @carocao-msft?